src/components/molecules/ChatPoll/ChatPoll.scss

```diff
@@ -11,6 +11,15 @@ button.ChatPoll__question {
   width: 100%;
   background-color: $secondary--light;
 
+  &:focus {
+    outline: none;
+  }
+
+  // This direct element style is used with RenderMarkdown
+  p {
+    margin-bottom: 0;
+  }
```

Comment:
What aspect of `RenderMarkdown` is causing you to need to do this? (I know it wraps basic text in `<p></p>`, but what is breaking because of that?)

Reply:
@0xdevalias please check comment https://github.com/sparkletown/sparkle/pull/1806#discussion_r668677323

Reply:
fixed in 3e4c5f3310eeb05fddf062dec5155638f3125926